lib/getResourceIdentifier/getResourceIdentifier.js

```diff
@@ -1,7 +1,13 @@
-export default (eresource = {}, type) => {
+export default (eresource = {}, type, returnAll) => {
   const identifiers = eresource?.identifiers || [];
-  // find first identifiers value of namespace 'type'
-  const value = identifiers.find(i => i.identifier?.ns.value === type)?.identifier?.value;
 
-  return value;
+  if (returnAll === true) {
+    // find all identifiers value of namespace 'type', return an array
+    const value = identifiers.filter(i => i.identifier?.ns.value === type).map(r => r?.identifier?.value);
```

Comment:
You can return this value immediately instead of assigning it to a variable. SonarLint should be catching this kind of thing if you have your IDE configured to show you those

Reply:
>  if you have your IDE configured to show you those

Can you tell me please how should the configuration look like?